## Welcome to the course "Bioinformatics and Microbiome Analysis (MB140P94)"

Offered by the Department of Genetics and Microbiology of the Charles University

**Course lecturers:** Tomáš Větrovský, Priscila Thiago, Iñaki Odriozola, Algora Camelia and Petr Baldrian

**Platform:** Git for online materials.

**Exams period:** June 2024

**Language:** English

**The classes are every Friday at 10:40 Central European Time (CET / GMT+1), in Viničná 7, B311.**

### Course introduction

In this course, we are going to explore the main approaches applied for microbiome studies based on high- throughput sequencing technologies. The students will get familiarized with methods for assessment of community composition, genome sequencing, metagenomics and metatranscriptomics. We are going to tackle subjects from the project planning and experimental design to the most used strategies for data treatment, data analysis, exploration and the presentation of results.

---

## Class Schedule

| Date      | Topic                                                         | Lecturer / Notes                   |
|-----------|---------------------------------------------------------------|------------------------------------|
| **7.3.**  | Introduction & History of sequencing and phylogenetics        |                                    |
| **14.3.** | Modern sequencing methods                                     |                                    |
| **21.3.** | Bioinformatic data and Linux (hands-on)                       | Linux                              |
| **28.3.** | Amplicon data (theory)                                        |                                    |
| **4.4.**  | Amplicon data (hands-on)                                      | SEED2 or Linux                     |
| **11.4.** | Methods for studying microbial communities – Stable Isotope Probing (SIP) | **Camelia Algora** (online)        |
| **18.4.** | **Good Friday Holiday (No Class)**                            |                                    |
| **25.4.** | Introduction to multivariate data analysis                    | **Iñaki Odriozola** (online)       |
| **2.5.**  | Shotgun data – genome assembly and metagenomics               |                                    |
| **9.5.**  | Experimental design                                           | **Petr Baldrian**                  |
| **16.5.** | Analysis of RNA and Metatranscriptomics                       | **Petr Baldrian**                  |

---

### 7.3. - Class 1: Introduction & History of sequencing and phylogenetics

[Class 1 Slides](https://raw.githubusercontent.com/VetrovskyTomas/MB140P94/main/2025/Class_1_Intro_and_History_of_sequencing_and_Phylogenetics.pdf)

### 14.3. - Class 2: Modern sequencing methods

[Class 2 Slides](https://raw.githubusercontent.com/VetrovskyTomas/MB140P94/main/2025/Class_2_Modern_sequencing_methods.pdf)

### 21.3. - Class 3: Bioinformatic data and Linux (hands-on)

[Class 3 Slides](https://raw.githubusercontent.com/VetrovskyTomas/MB140P94/main/2025/Class_3_BioinformaticLinux.pdf)

### 28.3. - Class 4: Amplicon data

[Class 4 Slides](https://raw.githubusercontent.com/VetrovskyTomas/MB140P94/main/2025/Class_4_Amplicon_data.pdf)

### 4.4. - Class 5: Amplicon data (hands-on)

[Linux metataxonomic pipeline](https://github.com/pdobbler/cool-python-scripts/tree/main/Amplicons/Course_MB140P94)

[SEED2 SOFTWARE](https://www.biomed.cas.cz/mbu/lbwrf/seed/)


### 11.4. - Class 6: Methods for studying microbial communities – Stable Isotope Probing (SIP)

[Class 6 Slides](https://raw.githubusercontent.com/VetrovskyTomas/MB140P94/main/2025/Class_6_Methods_for_studying_microbial_communities_SIP.pdf)

### 25.4. - Class 7: Introduction to multivariate data analysis

[Dunthorn_2017 - data in csv formate (click right and save as...)](https://raw.githubusercontent.com/VetrovskyTomas/MB140P94/main/2025/Dunthorn_2017.csv)

[origin - data in csv formate (click right and save as...)](https://raw.githubusercontent.com/VetrovskyTomas/MB140P94/main/2025/origin.csv)

[Class 7 Slides I.](https://raw.githubusercontent.com/VetrovskyTomas/MB140P94/main/2025/Class_7_Multivariate_Microbiome.pdf)  

[Class 7 Slides II.](https://raw.githubusercontent.com/VetrovskyTomas/MB140P94/main/2025/Class_7_Microbiome_course_2025.pdf)

### 2.5. - Class 8: Shotgun data – genome assembly and metagenomics

[Class 8 Slides](https://raw.githubusercontent.com/VetrovskyTomas/MB140P94/main/2025/Class_8_Shotgun_data_genome_assembly.pdf)  

### 9.5. - Class 9: Methods in Microbial Ecology - Experimental Design - Data Presentation

[Class 9 Slides](https://raw.githubusercontent.com/VetrovskyTomas/MB140P94/main/2025/Class_9_Methods_Experimental_design.pdf)  

### 16.5. - Class 10: Metatranscriptomics and analysis of microbial activity

[Class 10 Slides](https://raw.githubusercontent.com/VetrovskyTomas/MB140P94/main/2025/Class_10_Metatranscriptomics.pdf)  

  










